---
description: LOCALDB_ERROR_INTERNAL_ERROR
title: LOCALDB_ERROR_INTERNAL_ERROR | Документация Майкрософт
ms.custom: ''
ms.date: 03/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: performance
ms.topic: reference
ms.assetid: 60c7a47b-114c-441b-869e-9d8ebfe4148e
author: WilliamDAssafMSFT
ms.author: wiassaf
ms.openlocfilehash: 52653bc1610e27da8a8bcaf3e3420132d90eae4a
ms.sourcegitcommit: 0e0cd9347c029e0c7c9f3fe6d39985a6d3af967d
ms.translationtype: MT
ms.contentlocale: ru-RU
ms.lasthandoff: 12/02/2020
ms.locfileid: "96506208"
---
# <a name="localdb_error_internal_error"></a>LOCALDB_ERROR_INTERNAL_ERROR
 [!INCLUDE [SQL Server](../../includes/applies-to-version/sqlserver.md)]
    
## <a name="details"></a>Сведения  
  
| attribute | Значение |
| --------- | ----- |
|Название продукта|SQL Server|  
|Идентификатор события|264|  
|Источник события|Среда выполнения локальной базы данных SQL Server 12.0|  
|Компонент|API среды выполнения локальной базы данных|  
|Текст сообщения|Произошла непредвиденная ошибка внутри вызова метода API экземпляра локальной базы данных. Сведения об ошибке см. в журнале приложений Windows.|  
  
## <a name="explanation"></a>Объяснение  
 Произошла непредвиденная ошибка.  
  
## <a name="user-action"></a>Действие пользователя  
 Подробные сведения см. в журнале событий.  
  
  
